Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -984,31 +1003,32 @@ dma_tx_isr::
 #asm __root _dmaeth100_asmdebug
 ;------------------------------------------------------------------------------
 ; Transmit netport interrupt handler.
+; These interrupts can occur when data appears in the network peripheral.
+; DMA interrupts will occur when the DMA transfer is complete.
+; So, in theory, you could get this interrupt after or during a dma transfer
 ;------------------------------------------------------------------------------
 netport_tx_isr::
-#if 0
-	//// TEST
-	push	af
-   ioi ld a,(NBCSR)
-	pop	af
-	ipres
-	ret
-	//// END TEST
-#endif
+	   push  af
+	   push  bc
+	   push  de
+	   push  jkhl
+	   push  ix
+	   push  iy
+	   ld	hl, lxpc
+	   push	hl
+	   push	px
+	   push	py
+	   push	pz
 
-   push  af
-#ifdef DMAETH100_NET_STATS
-   push  bc
-   push  de
-   push  hl
-   push  ix
-  	ld    ix,_decb
-#endif
+   ld    ix,_decb
    ioi ld a,(NBCSR)
+; Not dorking with interrupts on the R6000

```

Comment:
Why shouldn't we change the interrupt priority on the Rabbit 6000?  Is it really that different from the 5000?